src/components/calcite-tip-group/calcite-tip-group.scss

```diff
@@ -1,10 +1,9 @@
 :host {
   @extend %component-host;
-  @extend %component-spacing;
   --calcite-tip-max-width: 540px;
```

Comment:
Is the tip group ever used outside of the tip manager? Could we update this to become an even int rem?